.env.example

```diff
@@ -25,6 +25,14 @@ REACT_APP_FANTOM_TESTNET_NODE_URL=""
 REACT_APP_POLYGON_NODE_URL=""
 REACT_APP_POLYGON_TESTNET_NODE_URL=""
 
+# Olympus Give
+# These 2 settings should be toggled to "true" if you want to use Olympus Give features
+REACT_APP_GIVE_ENABLED="false"
+REACT_APP_GIVE_GRANTS_ENABLED="false"
```

Comment:
In our prod env we are going to keep this as "false" for now, right?

Reply:
@0xJem can answer better, I have not been up to date on what the flags should be set to.

Reply:
Bad merge, I think. Should be true in the example file.

Prod also has true values for both, which I think is correct.